src/backend/package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "mapeo-mobile-node-next",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 3,
```

Comment:
just noting that the massive diff in this file (aside from removing eslint) is due to the lockfile version changing. I _think_ this is desirable in our case, as v3 is associated for npm v9 and later, which aligns with the versions of node that matter to us (18 + 20)

https://docs.npmjs.com/cli/v11/configuring-npm/package-lock-json#lockfileversion

not really sure why the lockfile version changed in this PR versus other PRs where we updated deps though 🤔